x/tradeshield/client/cli/tx.go

```diff
@@ -34,12 +34,14 @@ func GetTxCmd() *cobra.Command {
 	cmd.AddCommand(CmdUpdateSpotOrder())
 	cmd.AddCommand(CmdCancelSpotOrder())
 	cmd.AddCommand(CmdCancelSpotOrders())
+	cmd.AddCommand(CmdCancelAllSpotOrders())
```

Comment:
there CmdCancelSpotOrders and CmdCancelAllSpotOrders, whats the diff?

Reply:
CancelSpotOrders takes the orderIds to cancels the order, cancelAllSpotOrders simply cancels all pending orders for that users